tests/gui/components/drive-selector.spec.js

```diff
@@ -20,71 +20,17 @@ const _ = require('lodash')
 const m = require('mochainon')
 const angular = require('angular')
 require('angular-mocks')
+const utils = require('../../../lib/shared/utils')
 
 describe('Browser: DriveSelector', function () {
   beforeEach(angular.mock.module(
     require('../../../lib/gui/app/components/drive-selector/drive-selector')
   ))
 
   describe('DriveSelectorController', function () {
-    let $controller
-    let $rootScope
-    let $q
-    let $uibModalInstance
-    let WarningModalService
-
-    let controller
-
-    beforeEach(angular.mock.inject(function (
-      _$controller_,
-      _$rootScope_,
-      _$q_,
-      _WarningModalService_
-    ) {
-      $controller = _$controller_
-      $rootScope = _$rootScope_
-      $q = _$q_
-      $uibModalInstance = {}
-      WarningModalService = _WarningModalService_
-    }))
-
-    beforeEach(() => {
```

Comment:
Was this code all defunct then?

Reply:
It is now, but wasn't before. We needed access to the controller to access the memoize method, but now we don't.

Reply:
Meh :wink: